types/node/index.d.ts

```diff
@@ -1862,6 +1869,15 @@ declare module "os" {
     export function tmpdir(): string;
     export const EOL: string;
     export function endianness(): "BE" | "LE";
+    /**
+     * Gets the priority of thef process.
+     */
+    export function getPriority(): number;
+    /**
+     * Sets the priority of the process
+     * @param priority Must be in range of -20 to 19
+     */
+    export function setPriority(priority: number): number;
```

Comment:
There is also a variant available which receives `pid` as first argument.
Besides that the return type should be `void`.